package.json

```diff
@@ -9,6 +9,8 @@
                 "@babel/core": "7.25.2",
                 "@babel/runtime": "7.25.6",
                 "@fluentui/react": "8.120.7",
+                "@fluentui/react-components": "^9.55.1",
```

Comment:
Let's remove the carets (^) to avoid long pipeline runs

Reply:
Got it!